src/Authenticator/JwtAuthenticator.php

```diff
@@ -89,7 +89,7 @@ public function authenticate(ServerRequestInterface $request): ResultInterface
 
         $result = json_decode(json_encode($result), true);
 
-        $key = IdentifierInterface::CREDENTIAL_JWT_SUBJECT;
+        $key = $this->getConfig('tokenField', IdentifierInterface::CREDENTIAL_JWT_SUBJECT);
```

Comment:
This ties into the question why the config options are being flipped in the identifier... Should this use `dataField` instead?